jenkins/scripts/slurm_install.sh

```
@@ -25,11 +25,15 @@ slurm_install_setup() {
         python3 --version
         retry_command apt-get install -y libffi-dev
         nvidia-smi && nvidia-smi -q && nvidia-smi topo -m
-        if [[ $pytestCommand == *--run-ray* ]]; then
-            retry_command pip3 install --retries 10 "ray[default]==2.54.1"
+        if [[ "${SKIP_INSTALL:-0}" == "1" ]]; then
+            echo "SKIP_INSTALL=1: skipping pip installs (packages pre-installed in fat sqsh)"
+        else
+            if [[ $pytestCommand == *--run-ray* ]]; then
```

Shellcheck (0.11.0)

[warning] pytestCommand is referenced but not assigned.

(SC2154)